roles/kubespray-defaults/defaults/main/main.yml

```diff
@@ -548,9 +577,9 @@ ssl_ca_dirs: |-
 
 # Vars for pointing to kubernetes api endpoints
 kube_apiserver_count: "{{ groups['kube_control_plane'] | length }}"
-kube_apiserver_address: "{{ ip | default(hostvars[inventory_hostname]['fallback_ip']) }}"
+kube_apiserver_address: "{{ vars['ip' + default_net_mode] | default(hostvars[inventory_hostname]['fallback_ip' + default_net_mode]) }}"
```

Comment:
https://github.com/ansible/ansible/issues/15753#issuecomment-457641477
vars is not supported by ansible, it's internal. You may want to use `ansible.builtin.vars` lookup